chess/xboard.py

```diff
@@ -133,20 +133,20 @@ class PostHandler(object):
     >>> engine.go()
     e2e4
     >>>
-    >>> # Retrieve the score of the mainline (PV 1) after search is completed.
+    >>> # Retrieve the score of the mainline (PV1) after search is completed.
     >>> # Note that the score is relative to the side to move.
     >>> post_handler.post["score"]
     34
 
     See :attr:`~chess.xboard.PostHandler.post` for a way to access this dictionary
     in a thread-safe way during search.
 
-    If you want to be notified whenever new information is available
-    you would usually subclass the *InfoHandler* class:
+    If you want to be notified whenever new information is available,
+    you would usually subclass the :class:`~chess.xboard.InfoHandler` class:
```

Comment:
Since you're doing these changes, could you also make it `chess.xboard.PostHandler